pkg/resources/opnicluster/elastic/indices/opni_settings.go

```diff
@@ -24,11 +24,14 @@ const (
 	metricIndexTemplateName      = "opni-metric_rollover_mapping"
 	normalIntervalIndexName      = "opni-normal-intervals"
 	kibanaDashboardVersionDocID  = "latest"
-	kibanaDashboardVersion       = "v0.4.0"
+	kibanaDashboardVersion       = "v0.5.0"
 	kibanaDashboardVersionIndex  = "opni-dashboard-version"
 )
 
 var (
+	OldIndexPrefixes = []string{
+		"logs-v0.1.3*",
```

Comment:
is v0.1.3 the only version that needs upgrade? or this logic should probably do upgrade for any version that older than current version.

Reply:
It's the only version that will exist